tests/cpp/common/test_json.cc

```diff
@@ -333,18 +351,19 @@ TEST(Json, LoadDump) {
   Json::Dump(origin, &fout);
   fout.close();
 
-  buffer = common::LoadSequentialFile(path);
```

Comment:
I notice that `common::LoadSequentialFile()` has a special logic for POSIX target. Is there significant speed-up when you use `posix_fadvise()` followed by POSIX `read()`?

Reply:
@hcho3 I will remove it in next PR as it's not really useful for loading model from memory buffer, and conflicts with rabit check point logic.

But to answer your question, yes, it's a huge speed up.

Reply:
@hcho3 Looking into those tests, JSON number is much simpler.  Like not having `f` suffix, no `inf` or `NAN`.

![number](https://user-images.githubusercontent.com/16746409/62516778-f234e700-b7f3-11e9-9e75-a57730d88b0a.png)

Reply:
@trivialfis Interesting, so NAN and INF are not allowed in JSON. That's awesome